go.mod

```diff
@@ -30,19 +30,21 @@ require (
 	github.com/tendermint/tendermint v0.34.13
 	github.com/tendermint/tm-db v0.6.4
 	github.com/tyler-smith/go-bip39 v1.1.0
+	github.com/vulcanize/dxns v0.0.0-20210917114546-c736a2908785
```

Comment:
This doesn't look right - importing from the other repo?

Reply:
This was due to some utility functions defined in DXNS that we were using directly. I have migrated those utils to Ethermint now.